### PR TITLE
[TW-353] Signing in to SSO refresh

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Removed deprecated key-values from frontmatter
* Updated UI instructions and screenshots
* Updated terms and styling to be consistent throughout the page (and SSO section)
* Updated sign in instructions on other pages to be consistent throughout the documentation
* Ran Vale on the page, and found no major issues

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
Changes look great! 🌟

File: src/pages/docs/administration/sso/user-sso.md [hunk 1]
```diff
@@ -14,40 +11,38 @@ contextual_links:
...
+1. Enter the team domain, then select **Continue**.
 
-<img src="https://assets.postman.com/postman-docs/postman-sign-in-sso-v9.16.jpg" alt="Sign in with SSO" width="350px"/>
+    > Select **Stay signed in for 30 days** if you want to remain signed in after your current session for 30 days before re-authenticating. If you don't want to remain signed in on the computer you are working on, clear this checkbox. Note that you will be prompted to sign in again after 30 minutes.
 
-Postman will redirect you to the configured SSO provider. Enter your SSO details to sign in.
+    <img src="https://assets.postman.com/postman-docs/postman-sign-in-sso-v9.16.jpg" alt="Sign in with SSO" width="350px"/>
 
-> The first time you sign in to Postman, you may need to carry out extra steps to set up access to your identity provider account. Sign in using SSO and follow your provider's steps to connect the account to Postman.
+1. Select the [configured authentication method](/docs/administration/sso/admin-sso/#configuring-single-sign-on) you want to sign in with.
```
Comment (claireriley) [inline src/pages/docs/administration/sso/user-sso.md:42]: This link reads a little oddly to me if the majority of readers are users without access to configuring SSO. If I remember correctly, users can only select configured methods once they submit their team domain (not sure if still true?)
```suggestion
1. Select the authentication method you want to sign in with.
```

**Review by claireriley — APPROVED** [review]
Looks great! ⭐ one comment to consider

**Review by PriyambadaMadala — APPROVED** [review]

File: src/pages/docs/administration/sso/user-sso.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/user-sso.md]: Updated! Thanks!

Regarding your question, it's my understanding that users must enter the team domain before they can select an authentication method.

File: src/pages/docs/getting-started/postman-account.md [hunk 2]
```diff
@@ -77,13 +77,13 @@ During the sign up process, you have the option to create or join a [team](/docs
...
 
 > You can opt out of the sign in process at any time by going back to Postman and selecting **Skip and take me to Postman Desktop App**.
 
-Sign in to Postman in your browser by entering your account credentials or signing in with Google. If you're signing in to a Postman Professional or Enterprise team with SSO enabled, opt to **Sign in with SSO**.
+Sign in to Postman in your browser by entering your account credentials or signing in with Google. If you're signing in to a Postman Professional or Enterprise team with single-sign on (SSO) enabled, opt to **Sign in with SSO**. To learn more, see [Signing in to an SSO team](/docs/administration/sso/user-sso/).
 
 Select **Stay signed in for 30 days** if you want to remain signed in after your current session for 30 days before re-authenticating. If you don't want to remain signed in on the computer you're working on, clear this checkbox. Note that Postman will prompt you to sign in again after 30 minutes.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/getting-started/postman-account.md:88]: Select **Stay signed in for 30 days** if you want to remain signed in to your current session for 30 days before re-authenticating. "remain signed in **after** your current session" sounds off to me.

File: src/pages/docs/administration/sso/user-sso.md [hunk 3]
```diff
@@ -14,40 +11,38 @@ contextual_links:
...
+    <img src="https://assets.postman.com/postman-docs/postman-sign-in-sso-v9.16.jpg" alt="Sign in with SSO" width="350px"/>
 
-> The first time you sign in to Postman, you may need to carry out extra steps to set up access to your identity provider account. Sign in using SSO and follow your provider's steps to connect the account to Postman.
+1. Select the authentication method you want to sign in with.
 
-When you sign in, you will get a confirmation in your browser and you will be redirected back to Postman.
+1. Postman will redirect you to the configured IdP. Enter your SSO details to sign in.
 
-<img src="https://assets.postman.com/postman-docs/youre-signed-in-confirmation.jpg" width="350px" alt="Signed in confirmation"/>
+    > The first time you sign in to Postman, you may need to carry out extra steps to set up access to your IdP account. Sign in using SSO and follow your provider's steps to connect the account to Postman.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/sso/user-sso.md:46]: These may not be the provider's steps and may be steps that Postman wants the user to take such as linking the user's existing account.

Can we change this to:

...using SSO and following the steps to connect the account to Postman.

**Review by ArjunSingh-PM — COMMENTED** [review]
The changes look great. I have added a couple of comments for review.

File: src/pages/docs/administration/sso/user-sso.md (hunk 3 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/user-sso.md:46]: Thanks for your feedback. I created a new Jira ticket for looking into your feedback: https://postmanlabs.atlassian.net/browse/TW-968

File: src/pages/docs/getting-started/postman-account.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/postman-account.md:88]: Thanks for your feedback. I created a new Jira ticket for looking into your feedback: https://postmanlabs.atlassian.net/browse/TW-968

**Pull request merged**